### ISSUE TITLE
Forgeries

### ISSUE THREAD

**Issue by azaroth42** [linked-art/linked.art#369]
The use case:  An object created by an unknown person, who is forging the work of a more well known artist.

AAT has a "forgery" term, which is about the object, and a "copyist of" term which is about the production of an object, but no "forging of an object intended to look the same as an object by" (cough) term ... perhaps for obvious reasons.

Given this, I think that "forgery" is a classification on the object, and we can use "copyist of" for the relationship between the production of the object and the artist that the forger was copying.  This would follow the "style of" / "manner of" pattern

Alternatively, we could ask for an attribution qualifier for forgeries in AAT?

**Comment by edgartdata** [comment]
How about using https://www.getty.edu/vow/AATFullDisplay?find=&logic=AND&note=&subjectid=300137947 ?

I don't think 'copyist of' has a criminal intention. Beind a copyist was a perfectly legitimate occupation at a point in time.

**Comment by azaroth42** [comment]
I agree that copyist isn't the same as forger, certainly ... but I don't think 300137947 is the right term.

Looking again, there's also: http://www.getty.edu/vow/AATFullDisplay?find=&logic=AND&note=&page=1&subjectid=300252887 
Which isn't an attribution qualifier (like 'in the manner of') but at least is about the process / technique, so is closer.

The challenge (as always!) is to overlay a general thesaurus onto a specific data model, when the two were developed in isolation.
(And to bang the #186 drum until it breaks ... the need to make and publish decisions to save others the time and effort!)

I think there's two cases to think through, however:

1.  The object is a copy of another specific object; the deception is that it is the other object.
2. The object is an attempt to reproduce the style of an artist; the deception is that the object was produced by the artist.

I think we have the ontology and vocabulary for the first, but I'm not sure about the second.

**Comment by beaudet** [comment]
We will have “copyists” who come to paint in our galleries 😊

**Comment by azaroth42** [comment]
#186?

**Comment by azaroth42** [comment]
Propose defer
